src/secrets/components/SecretsTab.tsx

```diff
@@ -158,4 +154,11 @@ const SecretsTab: FC = () => {
   )
 }
 
-export default SecretsTab
+const mdtp = {
+  onShowOverlay: showOverlay,
+  onDismissOverlay: dismissOverlay,
+}
+
+const connector = connect(null, mdtp)
```

Comment:
remove this connector pattern in favor of `useDispatch`

Reply:
done